config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root "homes#index"
+  root "monsters#index"
+  resources :monsters
```

Comment:
Doesn't look like we have any ERB templates being used for monsters, so we don't need to have a regular controller, just the API for monsters used by React

Reply:
Implemented; erb templates removed